packs/unwanted-chrome-extensions.conf

```diff
@@ -0,0 +1,65 @@
+{
+  "platform": "windows,darwin",
+  "queries": {
+    "BetternetVPN": {
+      "query": "SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='gjknjjomckknofjidppipffbpoekiipm';",
+      "interval": 86400,
+      "description": "https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/",
```

Comment:
Would you mind wrapping the URLs to additional information in parens [similar to the mac attack](https://github.com/facebook/osquery/blob/master/packs/osx-attacks.conf#L14) packs?